mimic-iii/buildmimic/duckdb/import_duckdb.py

```diff
@@ -0,0 +1,378 @@
+    parser.add_argument('--mimic-code-root', help="location of the mimic-code repo (used to find concepts SQL)", default='../../../')
+    parser.add_argument('--make-concepts', help="generate the concepts views", action="store_true")
+    parser.add_argument('--skip-tables', help="don't create schema or load data (they must already exist)", action="store_true")
+    parser.add_argument('--skip-indexes', help="don't create indexes (implied by --skip-tables)", action="store_true")
+    parser.add_argument('--schema-name', help="put all object (except ccs_dx) into a schema (like the PostgreSQL version)", default=None)
+    args = parser.parse_args()
+    output_db = args.output_db
+    mimic_data_dir = args.mimic_data_dir
+    make_concepts = args.make_concepts
+    mimic_code_root = args.mimic_code_root
+    skip_tables = args.skip_tables
+    skip_indexes = args.skip_indexes
+    #TODO: validate schema_name is valid identifier
+    schema_name = args.schema_name
+
+    if not skip_tables:
+
+        connection = duckdb.connect(output_db)
+        print("Connected to duckdb...")
+
+        try:
+            schema_prequel = ""
+            if schema_name is not None:
+                connection.execute(f"CREATE SCHEMA IF NOT EXISTS {schema_name};")
+                connection.execute(f"USE {schema_name};")
+                schema_prequel = f"USE {schema_name};"
+
+            print("Creating tables...")
+    
+            with open(os.path.join(mimic_code_root, 'mimic-iii','buildmimic','duckdb','duckdb_add_tables.sql'), 'r') as fp:
+                sql = fp.read()
+                connection.execute(sql)
+
+            print("Loading data...")
+            
+            for f in os.listdir(mimic_data_dir):
+                m = re.match(r'^(.*)\.csv(\.gz)*', f)
+                if m is not None:
+                    tablename = m.group(1).lower()
+                    tables_in_schema.add(tablename)
+                    tablename = tablename if schema_name is None else schema_name+'.'+tablename
+                    print(f"  {tablename}")
+                    connection.execute(f"COPY {tablename} from '{os.path.join(mimic_data_dir,m.group(0))}' (FORMAT CSV, DELIMITER ',', HEADER);")
+            
+            if not skip_indexes:
+                print("Adding indexes...")
+        
+                with open(os.path.join(mimic_code_root, 'mimic-iii','buildmimic','duckdb','duckdb_add_indexes.sql'), 'r') as fp:
+                    sql = fp.read()
+                    connection.execute(sql)
+
+            print("Running checks...")
+    
+            with open(os.path.join(mimic_code_root, 'mimic-iii','buildmimic','duckdb','duckdb_checks.sql'), 'r') as fp:
+                sql = fp.read()
+                result = connection.execute(sql).fetchall()
+                for row in result:
+                    print(f"{row[0]}: {row[2]} records ({row[1]} expected) - {row[3]}")
```

Comment:
I was thinking about `--skip-checks`... should I just make `--run-checks` (off by default)? Don't love the idea of another set of static values...though I suppose they wouldn't change much.

Reply:
I ended up with a `validate.sql` and a `validate_demo.sql` with MIMIC-IV fwiw

Reply:
Though again, the row counts are something that screams "this should be a json somewhere else in the repo", rather than hard-coded into all these destined to rot SQL files